Sources/TestDRSMacros/Mocking/AddMockMacroExpansion.swift

```diff
@@ -166,7 +171,7 @@ public struct AddMockMacro: PeerMacro {
             named: className,
             inheritanceClause: inheritanceClause,
             members: classDeclaration.memberBlock.members,
-            config: .init(typeBeingMocked: .class, isPublic: classDeclaration.isPublic)
+            config: .init(typeBeingMocked: .class, isPublic: classDeclaration.isPublic, inheritsFromNSObject: false)
```

Comment:
Seems like this doesn't quite make sense as a class could actually inherit from `NSObject`. Maybe the name of this argument just needs to be changed.